Gemfile

```diff
@@ -30,6 +30,7 @@ gem "puma", "~> 4"
 gem "rails", "~> 6.1"
 gem "rswag-ui"
 gem "sassc", "~> 2.4.0"
+gem "stimulus-rails"
```

Comment:
There are a few ways to add stimulus to a Rails 6 project, but Rails 7 used the gem by default so I think it's probably the way to go.